api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -219,6 +219,8 @@ message Bootstrap {
     (udpa.annotations.security).configure_for_untrusted_upstream = true
   ];
 
+  BufferFactoryConfig buffer_factory_config = 33;
```

Comment:
Needs comment. Should this be placed in some other message like `ResourceTrackingConfig` for future proofing? Not sure what else we anticipate here, but not a big fan of tons of knobs on the top-level bootstrap.